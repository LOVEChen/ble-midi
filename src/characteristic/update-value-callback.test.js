/* eslint-env jest */
jest.mock('../util/logger');

test('should updateValueCallback()', () => {
  const mockLog = require('../util/logger').default;
  const updateValueCallback = require('./update-value-callback').default;
  updateValueCallback();
  expect(mockLog.mock).toMatchSnapshot();
  expect(mockLog.mock.calls.length).toEqual(1);
  expect(mockLog.mock.calls[0].length).toEqual(1);
});
