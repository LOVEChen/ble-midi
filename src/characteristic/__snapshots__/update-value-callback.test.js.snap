// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should updateValueCallback() 1`] = `
Object {
  "calls": Array [
    Array [
      "updateValueCallback",
    ],
  ],
  "instances": Array [
    undefined,
  ],
}
`;
